Dockerfile

```diff
@@ -3,16 +3,24 @@
 ARG TARGET=enduro
 
 FROM golang:1.20.5-alpine AS build-go
+LABEL maintainer="Artefactual Systems Inc <info@artefactual.com>"
+USER root
 WORKDIR /src
 ENV CGO_ENABLED=0
 COPY --link go.* ./
 RUN --mount=type=cache,target=/go/pkg/mod go mod download
 COPY --link . .
 
 FROM build-go AS build-enduro
+ARG VERSION_PATH
+ARG VERSION_LONG
+ARG VERSION_SHORT
+ARG VERSION_GIT_HASH
 RUN --mount=type=cache,target=/go/pkg/mod \
 	--mount=type=cache,target=/root/.cache/go-build \
-	go build -o /out/enduro .
+    go build -trimpath -o /out/enduro \
```

Comment:
I've noticed that you're not passing the new version flags to enduro-a3m-worker, why is that?

Reply:
I should definitely do that nice catch